script.js

```diff
@@ -1,53 +1,154 @@
-
-Now you can type "scissors" "paper" or "stone" to play another round!`;
+<br>${customPhrase}.
+<br>You've won ${timesPlayerWon}/${totalMatches} matches with ${numberOfDraws} draws.
+<br> Type in something other than "scissors", "paper","stone" ,"normal","reversed" or "korean" to set/change your username.
+<br>Now you can type "scissors" "paper" or "stone" to play another round!
+<br>You can also type in "normal", "reversed" or "korean" to change your game mode.(Your current game mode is ${gameMode})
+<br> If you would like the computer to play for you , leave the input box blank`;
+    return Message;
+  };
+  //for username
   if (
     input != "scissors" &&
     input != "paper" &&
     input != "stone" &&
-    input != "reversed stone" &&
-    input != "reversed paper" &&
-    input != "reversed scissors"
+    input != "normal" &&
+    input != "reversed" &&
+    input != "korean"
   ) {
-    myOutputValue = `You've entered an invalid option. Please type 'scissors'
-    , 'paper', 'stone', 'reveresed paper', 'reversed stone' or 'reversed scissors' to try again`;
+    userName = input;
+    myOutputValue = `Your username has been set to ${userName}. <br>Please type 'normal', 'korean' or 'reversed' to play the game(the gamemode has been set to normal by default) or type 'scissors', 'paper' or stone' to start the game`;
   }
-  if (
-    (input == `scissors` && elementfinal == "scissors ✂️") ||
-    (input == `paper` && elementfinal == "paper 📃") ||
-    (input == `stone` && elementfinal == "stone 💎") ||
-    (input == `reversed scissors` && elementfinal == "scissors ✂️") ||
-    (input == `reversed paper` && elementfinal == "paper 📃") ||
-    (input == `reversed stone` && elementfinal == "stone 💎")
-  ) {
-    myOutputValue = `The computer chose ${elementfinal} .
-You chose ${input}.
-
-It's a draw! Bummer.
-
-Now you can type "scissors" "paper" or "stone" to play another round!`;
+  if (input == `computer`) {
+    myOutputValue = `Your username has been set to computer and the computer will now play on your behalf. <br>Please type 'normal', 'korean' or 'reversed' to play the game(the gamemode has been set to normal by default)`;
   }
-  if (
-    (input == `scissors` && elementfinal == "paper 📃") ||
-    (input == `paper` && elementfinal == "stone 💎") ||
-    (input == `stone` && elementfinal == "scissors ✂️") ||
-    (input == `reversed scissors` && elementfinal == "stone 💎") ||
-    (input == `reversed paper` && elementfinal == "scissors ✂️") ||
-    (input == `reversed stone` && elementfinal == "paper 📃")
-  ) {
-    myOutputValue = `The computer chose ${elementfinal} .
-You chose ${input}.
-
-You win! Hooray.
-
-Now you can type "scissors" "paper" or "stone" to play another round!`;
+  //for normal gamemode
+  if (gameMode == normal) {
+    if (
+      (input == `scissors` && elementfinal == "stone 💎") ||
+      (input == `paper` && elementfinal == "scissors ✂️") ||
+      (input == `stone` && elementfinal == "paper 📃")
+    ) {
```

Comment:
This if statements and all those below can use the checkValidInput function I mentioned above as well.